docs/api/context-bridge.md

```diff
@@ -61,6 +61,19 @@ The `contextBridge` module has the following methods:
 * `apiKey` string - The key to inject the API onto `window` with.  The API will be accessible on `window[apiKey]`.
 * `api` any - Your API, more information on what this API can be and how it works is available below.
 
+### `contextBridge.evaluateInMainWorld(evaluationScript)` _Experimental_
+
+<!-- TODO(samuelmaddock): add generics to map the `args` types to the `func` params  -->
+
+* `evaluationScript` Object
+  * `func` (...args: any[]) => any - A JavaScript function to evaluate. This function will be serialized which means
+      that any bound parameters and execution context will be lost.
+  * `args` any[] (optional) - The arguments to pass to the provided function. These arguments must
+      be JSON-serializable.
```

Comment:
I've been trying to think of a scenario where it'd be backward incompatible to initially implementing this with JSON and then later using `contextBridge` to send the values, but unless someone explicitly relied on an argument *not* JSON serializing well, I think it should be fine to start with JSON serialization and enhance it later.

Reply:
I should be able to give the context-bridged arguments a try so I'll update how that goes soon enough. Since it's an experimental API though, I think it's safe if we have to alter its behavior.

Reply:
Proxied arguments are now implemented in https://github.com/electron/electron/pull/44411/commits/7200ef9eee2b9e97ef73809f7be8268d40b211fb which removes the JSON-serializable limitation.